### PR TITLE
Allow '..' in blob filename

### PR DESCRIPTION
**Pull request by millerdev**
Ran into a filename containing `..` while migrating staging. Need this to continue the migration.

@snopoke cc @orangejenny

### CODE REVIEW

**Comment by czue** [comment]
yikes that is just evil

**Comment by millerdev** [comment]
yeah, kinda wish I had not put the attachment name in the identifier.

✅

File: corehq/blobs/s3db.py [hunk 1]
```diff
@@ -100,7 +100,10 @@ def get_path(self, identifier=None, bucket=DEFAULT_BUCKET):
 
 
 def safepath(path):
-    if path.startswith(("/", ".")) or ".." in path or not SAFENAME.match(path):
+    if (path.startswith(("/", ".")) or
+            "/../" in path or
+            path.endswith("/..") or
```
Comment (gcapalbo) [inline corehq/blobs/s3db.py]: i don't have too much context but do you also need to check if `path.startswith("../")`?

Comment (gcapalbo) [inline corehq/blobs/s3db.py]: never mind, it would be caught on the first line

**Pull request merged**